pds/contracts/PackNFT_AllDay.cdc

```diff
@@ -125,8 +125,17 @@ pub contract PackNFT: NonFungibleToken, IPackNFT {
             PackNFT.openRequest(id: self.id)
         }
 
-        init(initID: UInt64, commitHash: String, issuer: Address ) {
-            self.id = initID
+       destroy() {
+           let p <- PackNFT.packs.remove(key: self.id) ?? panic("no such pack")
+           PackNFT.totalSupply = PackNFT.totalSupply - (1 as UInt64)
```

Comment:
I know we no longer use this to calculate the id but to be on a safer side, let's not decrement it

Reply:
okay, we can leave it then